.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -160,6 +160,9 @@ const MAINTAINED_SERVER_VERSIONS = [
   { name: '80x-enterprise', version: '8.0.x-enterprise' },
 ];
 
+const LATEST_MAINTAINED_SERVER_VERSION =
+  MAINTAINED_SERVER_VERSIONS[MAINTAINED_SERVER_VERSIONS.length - 1];
```

Comment:
I don't really mind keeping it as `latest-stable`, but this will not be aligned with our support policy (and we know it will start causing issues soon), so makes sense to me to use `MAINTAINED_SERVER_VERSIONS` as a source